dali/operators/ssd/box_encoder.h

```diff
@@ -76,29 +77,27 @@ class BoxEncoder<CPUBackend>: public Operator<CPUBackend> {
 
  private:
   const float criteria_;
-  vector<BoundingBox> anchors_;
+  vector<Box<2, float>> anchors_;
```

Comment:
I'm against it - in this case it doesn't add anything apart from a broken promise of being able to support other dimensionalities... and to keep things simple and changes minimal, I'd recommend:
`using BoundingBox = Box<2, float>`
This has both explicit 2D and is flexible in case of (unlikely) upgrade to 3D.

Reply:
Sure thing.